requirements.txt

```diff
@@ -0,0 +1,5 @@
+pulumi>=3.0.0,<4.0.0
+pulumi-gcp>=6.0.0,<7.0.0
+requests
+cryptography
+#python-on-whales
```

Comment:
pulumi-command needs adding to the requirements list

Reply:
Added